website/content/en/highlights/2023-07-05-0-31-0-upgrade-guide.md

```diff
@@ -61,3 +65,13 @@ of the event should it be serialized to JSON.
 Measuring the events like this allows a consistent measurement to be applied
 across all components regardless of how the source or sink serializes the event
 when connecting to the external service.
+
+### Potentially impactful changes
+
+#### AWS S3 endpoint path changes {#s3-paths}
+
+The handling of AWS S3 endpoints for the `aws_s3` source and sink has changed due to an upgrade of
+the SDK that we use. For S3-compatible APIs like Cloudflare R2 you may find that you need to remove
+the bucket name from the endpoint, if you have it specified. For example
+``https://xxxxxxxxxxxxxxxxxxxxxxxxxxx.r2.cloudflarestorage.com/<bucket name>` should be written as
+``https://xxxxxxxxxxxxxxxxxxxxxxxxxxx.r2.cloudflarestorage.com``.
```

Comment:
Should we be explicit about where to specify the bucket name instead (e.g., `bucket` field)?

Reply:
The `bucket` is already a required field so I'm not sure if that needs to be stated here 🤔

Reply:
Ah, that's a good point 👍